website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,54 @@
+
+<!--truncate-->
+
+* `[expect, jest-mock, pretty-format]` Remove `ES5` build files with a new minimum of support of ES2015 (Node 8) which were only used for browser builds ([#9945](https://github.com/facebook/jest/pull/9945))
+* `[jest-config, jest-resolve]` Remove support for `browser` field ([#9943](https://github.com/facebook/jest/pull/9943))
+* TypeScript definitions requires a minimum of TypeScript v3.8 ([#9823](https://github.com/facebook/jest/pull/9823))
+
+With the above changes Jest 26 is now 4 MiB smaller than Jest 25.5.4 (53 → 49 MiB). Please keep in mind that many dependencies like Babel are likely already part of your project. Jest's own size was reduced by 1.2 MiB (4.3 -> 3.1 MiB).
+
+While this is a good start, it isn’t enough to make Jest meaningfully leaner. We are planning on gradually reducing Jest's and its dependency tree size by about 70%. Most of the changes to reduce Jest’s default setup will be small breaking changes and we want to bring the community with us as much as possible. Therefore we will make the following changes in Jest 27 and Jest 28 using a [“Tick-Tock" release process](https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model):
+
+* Jest 27 will ship with a new test runner (`jest-circus`) and the Node.js environment by default. `jest-jasmine2` and `jest-environment-jsdom` will still be bundled so users can keep using them by changing one-line each in the configuration.
+* Jest 28 will remove `jest-jasmine2` and `jest-environment-jsdom` from the default distribution of Jest. The packages will still be actively maintained as part of the Jest project and be published separately. Users will need to install these packages to use them.
+
+Upon upgrading to these major releases, Jest will guide you through the necessary changes. If you’d like to get ahead and migrate to the new defaults now, and your project does not require a DOM environment, you can upgrade to Jest 26 and add the following configuration options:
+
+```json
+{
+  "jest": {
+    "testEnvironment": "node",
+    "testRunner": "jest-circus/runner"
+  }
+}
+```
+
+Jest will continue to ship with `babel-jest` enabled by default. It currently powers a few Jest features beyond compiling modern JS syntax to something understood by current versions of Node.js and browsers. It also powers Jest's code coverage and mocking of ES modules. Jest currently ships with experimental support for V8 coverage and native support for ES Modules (more on that below!). It is not possible to mock static ES Modules per the specification without transforming the code first, so we will be encouraging patterns that work without the transformation we use Babel for today. Once V8 coverage and native ESM support stabilizes in Jest, we will also be able remove `babel-jest` as a default but we will keep maintaining it.
+
```

Comment:
```suggestion
## New implementation of fake timers

Since 2017 we've been wanting to swap out Jest's implementation of fake timer with the more feature-complete implementation of Lolex - with Jest 26 this is finally happening! It took so long Lolex has even been renamed in the meantime to `@sinonjs/fake-timers`. It has taken so long since we wanted to make sure the implementation was solid and was usable for all the use cases Jest already supported. We're now confident it is stable enough for everybody to use.

The new implementation comes with some extra features, such as mocking `Date`, `queueMicrotask` and more, see their [readme](https://github.com/sinonjs/fake-timers/blob/master/README.md) for more details. We're calling the APIs through the Jest APIs you already know, so keep calling `jest.runAllTimers()` etc. like you've been doing. The docs as always are available [here](https://jestjs.io/docs/en/timer-mocks) on our website.

While we are confident this implementation is ready for prime time, we don't want to force people to rewrite their tests if they depend on Jest's current implementation to work, so for Jest 26 this new implementation will remain _opt-in_. Activate it by calling `jest.useFakeTimers('modern')` or by passing `modern` as the option in your config if you currently have `fake` there.

We will continue to support the legacy implementation going forward, but we will swap the default over to the "modern" timers in Jest 27. At that time you can opt-in to the legacy implementation by specifying `jest.useFakeTimers('legacy')`. If you know _now_ that you want to keep using the legacy implementation, you can specify that today and not have to change anything when Jest 27 comes around.

Huge thanks to [Carl-Erik Kopseng](https://github.com/fatso83), [Benjamin Gruenbaum](https://github.com/benjamingr) and other maintainers of `@sinonjs/fake-timers` for their help and patience when working with us to make this happen!
```